src/Events/QR/QR_Code.php

```diff
@@ -114,14 +113,17 @@ public function add_admin_table_action( $actions, $post ) {
 	 * @return void
 	 */
 	public function add_qr_code_meta_box(): void {
-		add_meta_box(
-			'tec-events-qr-code',
-			esc_html__( 'QR Code', 'the-events-calendar' ),
-			[ $this, 'render_qr_code_meta_box' ],
-			TEC::POSTTYPE,
-			'side',
-			'default'
-		);
+		$screen = get_current_screen();
+		if ( ! $screen || 'add' !== $screen->action ) {
+			add_meta_box(
+				'tec-events-qr-code',
+				esc_html__( 'QR Code', 'the-events-calendar' ),
+				[ $this, 'render_qr_code_meta_box' ],
+				TEC::POSTTYPE,
+				'side',
+				'default'
+			);
+		}
```

Comment:
Invert and bail early.
```suggestion
		if ( $screen && 'add' === $screen->action ) {
			return;
		}

		add_meta_box(
			'tec-events-qr-code',
			esc_html__( 'QR Code', 'the-events-calendar' ),
			[ $this, 'render_qr_code_meta_box' ],
			TEC::POSTTYPE,
			'side',
			'default'
		);
```